app/src/test/java/org/simple/clinic/patient/PatientMocker.kt

```diff
@@ -82,7 +82,9 @@ object PatientMocker {
   ): PatientAddress {
     return PatientAddress(
         uuid = uuid,
+        houseAddress = null,
```

Comment:
Same as earlier comment about reading the values for these from method parameters and setting default values for them.